encode/src/main/scala/org/broadinstitute/monster/etl/encode/EncodeIngest.scala

```diff
@@ -66,9 +66,54 @@ object EncodeIngest {
     // Add processing steps between the read and write here.
     pipelineContext
       .jsonFile[JsonObject](parsedArgs.experimentsJson)
+      .map { jObj =>
```

Comment:
`jObj` is accurate, but it's redundant with the type. You could name it something like `experimentObj` to add info not captured by the type.